src/sections/Meshery/Features-section/index.js

```diff
@@ -68,13 +68,13 @@ const FeaturesSection = () => {
             <Col xs={12} xl={8} className="smp-section-caraousel">
               <Slider {...settings}>
                 <div>
-                  <img src={Slide1} alt="Slide 1" loading="lazy" />
+                  <img src={Slide1} alt="Slide 1" width="1918px" height="978px" loading="lazy" />
```

Comment:
Based on what are we setting these values?

Reply:
@Nikhil-Ladha These are the original size of the images. However, this change doesn't matter as I have made a new PR #3978 which replaces this img(gif) with a video and sets height, width = 100%